project/Build.scala

```diff
@@ -9,7 +9,7 @@ import com.typesafe.sbt.SbtGit._
 
 object Build extends Build {
 
-  val baseVersion = "4.0.0"
+  val baseVersion = "4.0.1"
```

Comment:
All removals are from #298, hence only removing dead code — sorry for the PR on top, can resubmit. But I'm still adding `globalSettings`, which arguably* breaks forward compatibility. So should it be 4.0.1, 4.1.0 or 5.0.0?

*Unlike for Scala, I never considered compatibility for plugins, and this isn't mentioned in http://www.scala-sbt.org/0.13/docs/Plugins-Best-Practices.html.